baseplate/context/sqlalchemy.py

```diff
@@ -79,6 +80,16 @@ def __init__(self, engine):
         event.listen(self.engine, "after_cursor_execute", self.on_after_execute)
         event.listen(self.engine, "dbapi_error", self.on_dbapi_error)
 
+    def report_runtime_metrics(self, batch):
```

Comment:
Can we add a docstring to each of these to document the different metrics it reports?